plugins/PostProcessingPlugin/scripts/DisplayInfoOnLCD.py

```diff
@@ -476,8 +592,63 @@ def message_to_user(self, speed_factor: float):
         if print_start_time != "":
             print_start_str = "Print Start Time................." + str(print_start_time) + "hrs"
         else:
-            print_start_str = "Print Start Time.................Now."
+            print_start_str = "Print Start Time.................Now"
         estimate_str = "Cura Time Estimate.........." + str(print_time)
         adjusted_str = "Adjusted Time Estimate..." + str(time_change)
         finish_str = week_day + " " + str(mo_str) + " " + str(new_time.strftime("%d")) + ", " + str(new_time.strftime("%Y")) + " at " + str(show_hr) + str(new_time.strftime("%M")) + str(show_ampm)
-        return finish_str, estimate_str, adjusted_str, print_start_str
\ No newline at end of file
+
+        # If there are pauses and if countdown is enabled, then add the time-to-pause to the message.
+        if bool(self.getSettingValueByKey("countdown_to_pause")):
+            num = 1
+            for layer in data:
+                for p_cmd in pause_cmd:
+                    if p_cmd in layer or "Do the actual pause" in layer:
+                        adjusted_str += "\n" + self._get_time_to_go(layer.split("TIME_ELAPSED:")[1].split("\n")[0]) + " ET from start to pause #"   + str(num)
+                        num += 1
+        return finish_str, estimate_str, adjusted_str, print_start_str
+
+    def _get_time_to_go(self, time_str: str):
+        alt_time = time_str[:-1]
+        hhh = int(float(alt_time) / 3600)
+        if hhh > 0:
+            hhr = str(hhh) + "h"
+        else:
+            hhr = ""
+        mmm = ((float(alt_time) / 3600) - (int(float(alt_time) / 3600))) * 60
+        sss = int((mmm - int(mmm)) * 60)
+        mmm = str(round(mmm)) + "m"
+        time_to_go = str(hhr) + str(mmm)
+        if hhr == "": time_to_go = time_to_go + str(sss) + "s"
+        return time_to_go
```

Comment:
Something like this might be easier to read and update in future.
```suggestion
        alt_time = time_str[:-1]
        total_seconds = float(alt_time)
        hours = int(total_seconds // 3600)
        minutes = int((total_seconds % 3600) // 60)
        seconds = int(total_seconds % 60)

        time_to_go = f"{hours}h" if hours > 0 else ""
        time_to_go += f"{minutes}m"
        if hours == 0:
            time_to_go += f"{seconds}s"
        return time_to_go
```

Reply:
Done